apps/framework-docs/llm-docs/python/table-setup.md

```
@@ -232,6 +232,76 @@ time_series_table = OlapTable[TimeSeriesData](
 )
 ```
 
+## Externally Managed Tables
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)